tests/SwiftUI.Tests.FSharp/SwiftUI.Tests.FSharp.fsproj

```diff
@@ -12,12 +12,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="..\..\native\$(Configuration)\$(TargetFramework)-macos10.15\libSwiftUIGlue.dylib" />
+    <None Include="..\..\native\$(Configuration)\$(MacTargetFramework)\libSwiftUIGlue.dylib" />
     <Compile Include="Subjects\ViewWithOptionState.fs" />
     <Compile Include="TypeTests.fs" />
     <Compile Include="ColorTests.fs" />
     <Compile Include="Program.fs" />
-    <None Include="..\..\native\$(Configuration)\$(TargetFramework)-macos10.15\libSwiftUIGlue.dylib" Visible="false">
+    <None Include="..\..\native\$(Configuration)\$(MacTargetFramework)\libSwiftUIGlue.dylib" Visible="false">
```

Comment:
I didn't know that $(MacTargetFramework) existed. I searched. Anyway, good to know.